prime-router/src/main/kotlin/ActionMessages.kt

```diff
@@ -1,13 +1,55 @@
 package gov.cdc.prime.router
 
+/**
+ * Possible error codes when parsing/processing messages. Includes Hl7, CSV, FHIR, et al.
+ */
+enum class ErrorCode {
+    INVALID_MSG_PARSE_GENERAL,
+    INVALID_MSG_PARSE_TEXT,
+    INVALID_MSG_PARSE_TEXT_OR_BLANK,
+    INVALID_MSG_PARSE_NUMBER,
+    INVALID_MSG_PARSE_DATE,
+    INVALID_MSG_PARSE_DATETIME,
+    INVALID_MSG_PARSE_DURATION,
+    INVALID_MSG_PARSE_CODE,
+    INVALID_MSG_PARSE_CODE_ALT_VALUES,
+    INVALID_MSG_PARSE_CODE_VALUES,
+    INVALID_MSG_PARSE_ELEMENT_CODE,
+    INVALID_MSG_PARSE_TABLE,
+    INVALID_MSG_PARSE_TABLE_OR_BLANK,
+    INVALID_MSG_PARSE_EI,
+    INVALID_MSG_PARSE_HD,
+    INVALID_MSG_PARSE_ID,
+    INVALID_MSG_PARSE_ID_CLIA,
+    INVALID_MSG_PARSE_ID_DLN,
+    INVALID_MSG_PARSE_ID_SSN,
+    INVALID_MSG_PARSE_ID_NPI,
+    INVALID_MSG_PARSE_STREET,
+    INVALID_MSG_PARSE_STREET_OR_BLANK,
+    INVALID_MSG_PARSE_CITY,
+    INVALID_MSG_PARSE_POSTAL_CODE,
+    INVALID_MSG_PARSE_PERSON_NAME,
+    INVALID_MSG_PARSE_TELEPHONE,
+    INVALID_MSG_PARSE_EMAIL,
+    INVALID_MSG_PARSE_BLANK,
+    INVALID_MSG_PARSE_UNKNOWN,
+    INVALID_MSG_MISSING_FIELD,
+    INVALID_MSG_EQUIPMENT_MAPPING,
+    INVALID_HL7_MSG_VALIDATION,
+    INVALID_HL7_MSG_TYPE_MISSING,
+    INVALID_HL7_MSG_TYPE_UNSUPPORTED,
+    INVALID_HL7_MSG_FORMAT_INVALID,
+    UNKNOWN
```

Comment:
Action messages will default to UNKNOWN error code instead of empty/null/undefined